vizro-core/docs/pages/user-guides/assets.md

```diff
@@ -87,6 +87,71 @@ For reference, see the [Vizro CSS files](https://github.com/mckinsey/vizro/tree/
     [AssetsCSS]: ../../assets/user_guides/assets/css_change.png
 
 
+## Overwrite CSS properties for specific pages
+Each page is assigned a unique identifier (`page.id`), which is sanitized and then stored as the ID of an outer `<div>` element.
+This identifier is formatted according to CSS naming conventions by removing non-alphanumeric characters and replacing spaces with dashes.
+For instance, if a `page.id` is "Variable Analysis," it will be stored as a CSS ID of "variable-analysis."
+
+This unique identifier enables you to use CSS selectors to target specific pages and their components for styling.
+
+!!! note "Default Page ID"
+
+    If a page's ID is not explicitly set, it defaults to the page title.
+    Refer to our [user-guide on pages](pages.md) for more information on setting the page ID.
+
+
+Suppose you want to hide the page title on one page only. Here's how you can achieve this:
+
+1. Identify the CSS class or ID you need to target. For example, to hide the page title, you need to hide its parent container with the ID `right-header`.
+2. Use the unique identifier to target only the page you want to modify.
+
+
+!!! example "Hide page title on selected pages"
+    === "my_css_file.css"
+    ```css
+    #page-with-hidden-title #right-header {
+      display: none;
+    }
+    ```
+    === "app.py"
+        ```py
+        import vizro.models as vm
+        from vizro import Vizro
+
+        page_one = vm.Page(
+            title="Page with hidden title",
+            components=[vm.Card(text="""# Placeholder""")]
+        )
+
+        page_two = vm.Page(
+            title="Page with shown title",
+            components=[vm.Card(text="""# Placeholder""")]
+        )
+
+        dashboard = vm.Dashboard(pages=[page_one, page_two])
+        Vizro().build(dashboard).run()
+        ```
+    === "app.yaml"
+        ```yaml
+        # Still requires a .py to add data to the data manager and parse YAML configuration
+        # See yaml_version example
+        pages:
+        - components:
+            - text: |
+                # Placeholder
+              type: card
+          title: Page with hidden title
+        - components:
+            - text: |
+                # Placeholder
+              type: card
+          title: Page with shown title
+        ```
+    === "Result"
+         [![PageTitle]][PageTitle]
+
+    [PageTitle]: ../../assets/user_guides/assets/css_page_title.png
+
 
 ## Overwrite CSS properties in selective components
```

Comment:
Is this section still correct? e.g. it talks a lot about `_outer`, and I'm not sure whether that's still used much after your recent bootstrap refactorings?

Reply:
This still holds, but it gets a bit confusing now because it does not hold for all components. Let's discuss in our call, but just to show you what I mean:

Here we don't have the `_outer` suffix for an ID, because we can provide the self.id directly to the component that gets outputted:

`dbc.Button(id=self.id, children=self.text)`

However, here we still need the `_outer` suffix because of two reasons:

1. We once aligned that the `self.id` is being provided to the inner component that might be reactive to callbacks. In the example of the card it is the `dcc.Markdown` component. Because of that we cannot provide the `self.id` to the outermost component. 
2. Let's say someone wants to change the card background-color but only for a selective card and not for all. Here we need to provide an `id` to the outermost wrapper of the card so we can selectively target all inner components. And because we can't use `self.id`, we started using `self.id_outer`.

Happy to change this current system though, just not sure what's the best approach here 🤔 

`dbc.Card(card_content, className=card_class, id=f"{self.id}_outer")`

Reply:
Yeah, that is what I expected but I didn't know how prevalent the `_outer` suffix was any more.

I think the current system basically makes sense (using `self.id` for the "core" reactive component rather than the outer container) but we should _consider_ (but maybe not do...) removing the `_outer` ones if they're just there for CSS selectors [in favour of using `has()`](https://stackoverflow.com/questions/1014861/is-there-a-css-parent-selector).

Reply:
Yes, I like that! Let's discuss in our call later. But I would also be in favour of killing the entire _outer system and just updating the docs 👍